### DISCUSSION TITLE
Unable to login missing location redirect

### DISCUSSION THREAD

**Discussion by DimX00** [ianjwhite99/sync-connect-sdk#65] [Q&A] [answered]
Hi, When I try to connect to get the access token. I get the above error. Unable to login, missing location redirect. 

Any idea why this happens?

Thank you

**Comment by ianjwhite99** [comment, marked as answer]
> Hi, When I try to connect to get the access token. I get the above error. Unable to login, missing location redirect. 
> 
> 
> 
> Any idea why this happens?
> 
> 
> 
> Thank you

Are you running the latest version of the SDK? Also, are you sure your username and password is correct? Please verify that and let me know the result.

**Reply by DimX00** [reply]
Hi,

Yes, I am using v1.3.8 and I am using the correct username and password. I am able to login to FordPass with this login.

**Reply by DimX00** [reply]
I apologize it was the incorrect login. Now I am getting the following error: Error Fetching Access Token with PKCE Code

**Reply by DimX00** [reply]
I figured it out. I accidentally deleted the default ConnectedCar client_id. I am able to get the data now. Thank you for your help!

**Reply by ianjwhite99** [reply]
No problem. Glad you could get it solved.

**Reply by DimX00** [reply]
Do you know why it doesn't return mileage?

The only data that I get back is VIN, NickName, VehicleType, Color, make, model and year. Everything else is none or 0.

`'productVariant': None, 'averageMiles': None, 'estimatedMileage': '0', 'mileage': None, 'mileageDate': '01/01/0001 00:00:00', 'mileageSource': None, 'drivingConditionId': None, 'configurationId': None, 'primaryIndicator': '', 'licenseplate': None, 'purchaseDate': None, 'registrationDate': '2022-03-13T20:01:22.580Z', 'ownerCycle': None, 'ownerindicator': 'N`

I can see the mileage in the dashboard when I login.

**Reply by DimX00** [reply]
Sorry. Disregard the above question. I am able to get mileage by directly requesting for odometer as opposed to just using the vehicle details.
